frappe_s3_attachment/controller.py

```diff
@@ -87,6 +87,8 @@ def key_generator(self, file_name, parent_doctype, parent_name):
 
         doc_path = None
 
+        parent_doctype = parent_doctype if parent_doctype else "media"
```

Comment:
Why media? We can use File, File is already a doctype in frappe.

Reply:
Sure, we can use "file/File".  So do we close this and create a new one with "File"?

Reply:
Why `file/File` any specific reason?

Reply:
Didn't mean literally `file/File` just a distinction between the lowecase/uppercase "file"

Reply:
It should be File as we have File doctype in frappe

Reply:
Updated to `File`

Reply:
https://github.com/zerodha/frappe-attachments-s3/blob/master/frappe_s3_attachment/controller.py#L208
can you update this to `doc.get("attached_to_doctype", "File")` instead of adding inside the key generator method.